src/main/java/uk/gov/hmcts/reform/hmc/helper/hmi/ListingOtherConsiderationsMapper.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.hmc.helper.hmi;
+
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class ListingOtherConsiderationsMapper {
+
+    public List<String> getListingOtherConsiderations(Boolean hearingInWelshFlag,
+                                                      List<String> facilityTypes) {
+        List<String> listOtherConsiderations = new ArrayList<>();
+        if (null != hearingInWelshFlag) {
+            listOtherConsiderations.add(hearingInWelshFlag.toString());
+        } else {
+            listOtherConsiderations.add(Boolean.FALSE.toString());
+        }
```

Comment:
I don't think the ticket is clear enough about this one, but this logic doesn't sound right.

On our scope doc it says that we should be setting it to some system defined variable when `hearingInWelshFlag` is true. I don't think we should be setting it simply to the string value "true"! And when false it doesn't sound like we should be putting anything.

Could you catch up with the BAs on this please for clarification? The scope doc still has the value we should set it to as undefined!

Reply:
Query with Arup atm.

Reply:
"CY" value applied for now as advised.